### ISSUE TITLE
TypeScript typings

### ISSUE THREAD

**Issue by fandy** [peterpalau/react-bnb-gallery#14]
👋 Peter, I'm giving your lightbox another try. I noticed there were no TS typings out of the box so I added my own. Hope it's useful, and feel free to make any corrections:

```
declare module 'react-bnb-gallery' {
  import ReactBnbGallery from 'react-bnb-gallery'

  declare interface Props {
    // Initial photo index to show.
    activePhotoIndex: number
    // Called when a photo is pressed.
    activePhotoPressed: () => void
    // Called when left key of the keyboard is pressed.
    leftKeyPressed: () => void
    // Called when next control button is pressed.
    nextButtonPressed: () => void
    // Called when the gallery modal is going to close.
    onClose: () => void
    // The number of photos to preload on gallery initialization.
    preloadSize: number
    // Called when previous control button is pressed.
    prevButtonPressed: () => void
    // Array of photos. It can be an array of photos URLs or an array of objects. See the [photo object] props bellow.
    photos: Photo[] | string[]
    phrases?: object
    // Called when right key of the keyboard is pressed.
    rightKeyPressed?: () => void
    // Shows the modal when initialized.
    show?: boolean
    // Whether the gallery should show thumbnails.
    showThumbnails?: boolean
    // Whether the gallery should react to keyboard events.
    keyboard?: boolean
    // Whether the gallery should cycle continuously or have hard stops.
    wrap?: boolean
    // Sets the opacity level for the component.
    opacity?: number
    // Sets the background color of the gallery component.
    backgroundColor?: string
    // Specifies the stack order of the component.
    zIndex?: number
  }

  declare interface Photo {
    // The src attribute value of the image.
    photo: string
    // The current number of the photo.
    number?: number
    // Photo description.
    caption?: string
    // Photo secondary description, like the photo author or the name of the place where it was taken.
    subcaption?: string
    // The url of the photo thumbnail. The preferred size for each thumbnail is 100x67.
    thumbnail?: string
  }
}
```

**Comment by peterpalau** [comment]
Thank you so much for this @fandy

**Comment by peterpalau** [comment]
Solved via 188cf3c99fd7bebb83cca1028086deb4be41f49b.

Closing now
